doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+How to report a bug
+-------------------
+
+1. Set the system to Permissive and reproduce the issue (*)
+2. When reporting the issue, include the permissive audit logs as well
+3. The `audit2allow` output can be helpful to include in the report
+   when it's limited to the denials relevant to the issue, but it is
+   not enough on its own. It's essential to also include the actual AVC
+   denials (and ideally, the full permissive audit logs around the time
+   the issue is triggered.)
+
+If a bug doesn't already exist on Launchpad or Bugzilla, create a [RDO
+bug](https://bugzilla.redhat.com/enter_bug.cgi?product=RDO) with the
+`openstack-selinux` component. Having a bug number is necessary to add
+unit tests.
+
+(*) In Enforcing mode, SELinux stops at the first denial which can hide
+    more of them. Permissive mode enables us to see the full list of
+    AVC denials, so that they can be resolved all at once rather than
+    one at a time.
+
+How to run the tests
+--------------------
+
+1. Install the `selinux-policy-devel` package
+2. Ensure the path `/usr/share/openstack-selinux/master` exists
+3. From your local openstack-selinux repository, run the following
+   command as root:
+
+        $ make clean all install check
+
+It is recommended to use a VM for this.
```

Comment:
Do we want to mention that testing/development of SELinux policies cannot be done within a container?

Reply:
I'll add a follow-up PR for this.